tests/fixtures.py

```
@@ -0,0 +1,37 @@
+
+@pytest.fixture
+def authenticated_user(client, user):
+    """
+    Fixture to create and authenticate a user.
+    """
+    client.force_login(user)
+    return user
+
+
+@pytest.fixture
+def api_client(db):
```

Ruff (0.14.4)

Unused function argument: `db`

(ARG001)